src/Symfony/Bridge/PhpUnit/DeprecationErrorHandler/Configuration.php

```diff
@@ -165,6 +165,32 @@ public function tolerates(array $deprecationGroups)
         return true;
     }
 
+    /**
+     * @param array<string,DeprecationGroup> $deprecationGroups
+     *
+     * @return bool true if the thresholds is not reached for the deprecation type
+     */
+    public function toleratesGroup(string $groupName, array $deprecationGroups): bool
+    {
+        $grandTotal = 0;
+
+        foreach ($deprecationGroups as $type => $group) {
+            if ('legacy' !== $type) {
+                $grandTotal += $group->count();
+            }
+        }
+
+        if ($grandTotal > $this->thresholds['total']) {
+            return false;
+        }
```

Comment:
The description of the method should explain why it also checks the grand total. I wouldn't expect that check to be performed based on the current name.